### PR TITLE
Make it easier to change models used 

### LINKED ISSUE #78: Make it easier to change models used
# Summary
In the settings it should be easy to change the models used across the applications when e.g. changing language. Preferably in the settings. 

# Examples
Settings -> Set language -> Models are now updated 

# Context
Right now the models are hard-coded in the modules, it should be easier to change these when changing language or perhaps when changing to any other model.

### PR DESCRIPTION
**Pull request by Blinningjr**
# Proposed changes
* Implement the feature to change all nlp models used based on language using the cli.
* Change so that the nlp models are no longer reloaded every time a automation task is executed.

## How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.
- [x] Set `model_language` in `config/user.yaml` to an empty string and then start the cli. The user get a prompt that makes them choose a language from the available languages. 
- [x] Run the command `language` or `lang` in the cli. The user get a prompt that makes them choose a language from the available languages.
- [x] Run the command `help` or `h` and the new command should be listed.
- [x] Run all the demos.

# Related issue
closes #78

### CODE REVIEW

**Review by markhakansson — APPROVED** [review]
Nice, good job

File: lib/settings.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import os
 import yaml
-from lib.cli.config import config_user_name, config_email_address, config_email_host
+from lib.cli.config import config_user_name, config_email_address, config_email_host, config_model_language
```
Comment (Aleman778) [inline lib/settings.py:3]: Does the settings module really need to be connected to the CLI? I would like to reuse this for the GUI as well.
You don't need to fix this now but something to keep in mind.

**Review by Aleman778 — APPROVED** [review]
Great!

File: lib/settings.py (hunk 1 above)
Comment (Blinningjr) [reply lib/settings.py:3]: Yeh, I agree that it should be separate from the cli. But I think that that is out of the scope for this pr, will make a new issue for it.

Comment (Blinningjr) [reply lib/settings.py:3]: Made issue #111 to solve this.

**Pull request merged**